src/Features/CSharp/Portable/ConvertToRawString/ConvertRegularStringToRawStringCodeRefactoringProvider.cs

```diff
@@ -162,21 +163,30 @@ private static bool CanConvertStringLiteral(SyntaxToken token, out CanConvertPar
                 // This changes the contents of the literal, but that can be fine for the domain the user is working in.
                 // Offer this, but let the user know that this will change runtime semantics.
                 canBeMultiLineWithoutLeadingWhiteSpaces = token.IsVerbatimStringLiteral() &&
-                    HasLeadingWhitespace(characters) &&
+                    (HasLeadingWhitespace(characters) || HasTrailingWhitespace(characters)) &&
```

Comment:
discovered as we have a lots of tests that use:

```c#
@"class C
{
}
"
```

This allows the user to say they want to convert, while removing that trailing newline.

Reply:
I agree that we want to offer this fix, but I find it exceedingly unfortunate that it was applied in bulk #67059. I would have expected that bulk refactoring to avoid any behavior changes.

Reply:
I disagree. Applying in bulk without the behavior change just leads to horrendous outcomes with tests full of unnecessary whitespace that was not part of the original test's motivations. 

That whitespace was there because of how c# literals work, not because the test author wanted them. 

The conversion (which you have to opt into) does change the strict c# meaning of the strings. But it actually makes the tests reflect what the test author was intending.